packages/types/package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "scripts": {
     "build": "tsc -p tsconfig.json",
-    "dev": "tsc -p tsconfig.json --watch",
+    "dev": "tsc -p tsconfig.json --watch --preserveWatchOutput | awk '{gsub(/\\033c/,\"\") system(\"\")}1'",
```

Comment:
❓ would `awk ...` for something developing in OS other than Linux or MacOS (eg Microsoft)?